contracts/modules/TransferManager.sol

```diff
@@ -249,11 +240,11 @@ contract TransferManager is OnlyOwnerModule, BaseTransfer {
         bytes calldata _data
     )
         external
-        onlyWalletOwnerOrModule(_wallet)
+        onlyWalletOwnerOrFeature(_wallet)
         onlyWhenUnlocked(_wallet)
         onlyAuthorisedContractCall(_wallet, _contract)
```

Comment:
We need to update this modifier to check that we are not calling another Feature.